app/src/main/java/ar/com/wolox/android/example/ui/home/HomeFragment.kt

```diff
@@ -0,0 +1,48 @@
+package ar.com.wolox.android.example.ui.home
+
+import androidx.core.util.Pair
+import ar.com.wolox.android.R
+import ar.com.wolox.android.example.ui.home.news.NewsFragment
+import ar.com.wolox.android.example.ui.home.profile.ProfileFragment
+import ar.com.wolox.wolmo.core.adapter.viewpager.SimpleFragmentPagerAdapter
+import ar.com.wolox.wolmo.core.fragment.WolmoFragment
+import com.google.android.material.tabs.TabLayout
+import kotlinx.android.synthetic.main.fragment_home.*
+import javax.inject.Inject
+
+class HomeFragment : WolmoFragment<HomePresenter>(), HomeView {
+
+    @Inject internal lateinit var mPageNews: NewsFragment
```

Comment:
Remember not to use hungarian anotation. Use camelCase. In these cases: pageNews, pageProfile and fragmentHomePagerAdapter